sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/query.sql

```diff
@@ -344,6 +344,114 @@ metrics_org_mozilla_ios_fennec AS (
   WHERE
     mozfun.norm.truncate_version(client_info.app_display_version, 'major') >= 28
 ),
+-- metrics for Focus Android Baseline release
+metrics_org_mozilla_focus AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    client_info.client_id,
+    normalized_country_code AS country,
+    'Focus Android Baseline' AS app_name,
+    'Focus' AS normalized_app_name,
+    client_info.app_display_version AS app_version,
+    'Release' AS channel,
+    normalized_os AS os,
+    client_info.android_sdk_version AS os_version,
+    metrics.string.browser_default_search_engine AS default_search_engine,
+    CAST(NULL AS STRING) AS default_search_engine_submission_url,
+    sample_id,
+    metrics.labeled_counter.browser_search_search_count AS search_count,
+    metrics.labeled_counter.browser_search_ad_clicks AS search_ad_clicks,
+    metrics.labeled_counter.browser_search_in_content AS search_in_content,
+    metrics.labeled_counter.browser_search_with_ads AS search_with_ads,
+    client_info.first_run_date,
+    ping_info.end_time,
+    ping_info.experiments,
+    metrics.counter.browser_total_uri_count,
+    client_info.locale,
+  FROM
+    org_mozilla_focus.metrics AS org_mozilla_focus_metrics
+),
+-- metrics for Focus Android Baseline beta
+metrics_org_mozilla_focus_beta AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    client_info.client_id,
+    normalized_country_code AS country,
+    'Focus Android Baseline' AS app_name,
+    'Focus' AS normalized_app_name,
+    client_info.app_display_version AS app_version,
+    'Release' AS channel,
```

Comment:
```suggestion
    'Beta' AS channel,
```

Should this be Beta?

Reply:
Good catch, this is a templated query I'll update the template